test/jest-common.js

```diff
@@ -0,0 +1,13 @@
+const path = require('path');
+
+module.exports = {
+  rootDir: path.join(__dirname, '..'),
+  moduleNameMapper: {
+    '~/(.*)': '<rootDir>/$1',
```

Comment:
Please add [these lines](https://github.com/kawalcovid19/wargabantuwarga.com/pull/576/files#diff-d42b7bb72c8217948964f5095a1fdcc0393828107f34168dfc35a02ab9291ceb) to make it easier to test the `_app.tsx` file in the future. 😁 

```suggestion
    "\\.(css|less)$": "identity-obj-proxy",
     "typeface-inter": "identity-obj-proxy",
    '~/(.*)': '<rootDir>/$1',
```

You would also need to install the `identity-obj-proxy` as the dev dependency.